File: benchmarks/random-1M/main.go
package main

import "fmt"
import "time"
import "math/rand"

func main() {
	var before = time.Now().UnixNano()

	var n = 0.0
	for i := 0; i < 1_000_000; i++ {
		n = rand.Float64()
	}
	n = n

	var after = time.Now().UnixNano()

	fmt.Println(after - before)
}

File: benchmarks/random-1M/Main.java
public class Main {

    public static void main(String[] args) {
        long before = System.nanoTime();

        double n = 0;
        for (int i = 0; i < 1000000; i++) {
            n = Math.random();
        }

        long after = System.nanoTime();

        System.out.println(after - before);
    }

}